diff --git a/app/Http/Controllers/CartController.php b/app/Http/Controllers/CartController.php
index 64f8616c38caad7f7e94206f30421466069f1885..75a3f4fff8eda0fb567db5d7123da620583f302c 100644
--- a/app/Http/Controllers/CartController.php
+++ b/app/Http/Controllers/CartController.php
@@ -13,6 +13,7 @@ class CartController extends Controller
 	public function index()
 	{
 		$cart = Cart::content();
+
 		return view('cart', compact('cart'));
 	}
 
@@ -23,12 +24,14 @@ class CartController extends Controller
 		//request()->session()->flash('success', "$part->name added to cart!");
 		return redirect(url()->previous('/'));
 	}
+
 	public function remove(string $rowId)
 	{
 		Cart::remove($rowId);
 
 		return back();
 	}
+
 	public function update(request $request, $id)
 	{
 		Cart::update($id, $request->qty);
@@ -37,7 +40,7 @@ class CartController extends Controller
 	}
 
 	/**
-	 * Get cart item quantity
+	 * Get cart item quantity.
 	 *
 	 * Return the quantity of a cart item by model id
 	 *
@@ -50,8 +53,10 @@ class CartController extends Controller
 		if (request()->wantsJson()) {
 			$cart = Cart::content();
 			$item = $cart->where('id', $id)->first();
+
 			return $item ? (int) $item->qty : 0;
 		}
+
 		return abort(404);
 	}
 }
diff --git a/app/Http/Controllers/NewsletterController.php b/app/Http/Controllers/NewsletterController.php
index d4c9ec488e7164d47879f7ce3dc428efccc258d6..6eb9fa583f1090b56f11f5f9a6e249da78bc7916 100644
--- a/app/Http/Controllers/NewsletterController.php
+++ b/app/Http/Controllers/NewsletterController.php
@@ -11,7 +11,7 @@ class NewsletterController extends Controller
 {
 	public function store(Request $request)
 	{
-		if (!Newsletter::isSubscribed($request->email)) {
+		if (! Newsletter::isSubscribed($request->email)) {
 			Newsletter::subscribePending($request->email);
 
 			return redirect('/')->with('success', 'Thanks For Subscribe');
diff --git a/app/Nova/Part.php b/app/Nova/Part.php
index 7b65f6f3a4caf6a424f3d726535a51bff5ac7bd9..cf969e3893369bed3c8f0638c98e9b407314ddcf 100644
--- a/app/Nova/Part.php
+++ b/app/Nova/Part.php
@@ -55,7 +55,7 @@ class Part extends Resource
 	 * @var array
 	 */
 	public static $search = [
-		'id', 'title', 'description', 'sku'
+		'id', 'title', 'description', 'sku',
 	];
 
 	/**
@@ -104,8 +104,10 @@ class Part extends Resource
 	{
 		if ($request->user()->hasRole('Super Admin')) {
 			info('Admoun');
+
 			return $query;
 		}
+
 		return $query->where('user_id', auth()->id());
 	}
 
diff --git a/app/Providers/AppServiceProvider.php b/app/Providers/AppServiceProvider.php
index 733d59a87b16d0b5dcc6cf4c139ab37d07308aae..ee37b76e5042fe442228340bc204a0230e8337c1 100644
--- a/app/Providers/AppServiceProvider.php
+++ b/app/Providers/AppServiceProvider.php
@@ -8,10 +8,10 @@ use App;
 use App\Facades\WishlistFacade;
 use App\Nova\Templates\FooterOptions;
 use App\Nova\Templates\HeaderOptions;
-use Gloudemans\Shoppingcart\Facades\Cart;
 use Whitecube\NovaPage\Pages\Manager;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\ServiceProvider;
+use Gloudemans\Shoppingcart\Facades\Cart;
 
 class AppServiceProvider extends ServiceProvider
 {
diff --git a/routes/web.php b/routes/web.php
index f37b44bc2bf9c4016c36570c9628498451ea59d9..580ba5295a576b0ae71103d6c81397ccc50085bf 100644
--- a/routes/web.php
+++ b/routes/web.php
@@ -86,7 +86,6 @@ Route::get('/store/{store}/about', 'StoreAboutController@show')->name('store.abo
 Route::view('/pricing', 'pricing');
 Route::post('/newsletter', 'NewsletterController@store')->name('newsletter');
 
-
 Route::fallback('FallBackController');
 
 Route::get('/wishlist', 'WishlistController@index')->name('wishlist');
